### PR TITLE
grafana/ui: Introduced CallToActionCard

### PR DESCRIPTION
**Pull request by dprokop**
Driven by the needs of #15533 

**This work is based on https://github.com/grafana/grafana/pull/16228, merge that PR first!!!**

image

- Introduced more abstract and with simpler API `CallToActionCard` component to a grafana/ui that removes the need for `EmptyListCTA`
- `EmptyListCTA` is now refactored to use the new component underneath
- `EmptyListCTA` tests and css is removed ad `CallToActionCard` holds the tests end implements styles using Emotion ( introduced in https://github.com/grafana/grafana/pull/16228)
- Fixed `AlertTab` not displaying icon on the CTA button.

@Ijin08 would you mind doing a manual testing on that as well as overview of the styles? This uses new buttons and theme variables.

### CODE REVIEW

File: public/sass/components/_empty_list_cta.scss [hunk 1]
```diff
@@ -1,24 +0,0 @@
-.empty-list-cta {
```
Comment (torkelo) [inline public/sass/components/_empty_list_cta.scss]: this class is being used 4-5 other locations as well (not only from this react component), templating list, annotation links, dashboard links, manage dashboard, notifications list, playlists. 

Not sure why the react componet (and angular wrapper) is not being used, think for the annoations & templating use cases it did not work as we need custom body with help box but for the others not sure.

**Review by torkelo — CHANGES_REQUESTED** [review]

File: public/sass/components/_empty_list_cta.scss (hunk 1 above)
Comment (Ijin08) [reply public/sass/components/_empty_list_cta.scss]: Screenshot 2019-03-27 at 08 34 48

Comment (dprokop) [reply public/sass/components/_empty_list_cta.scss]: > Not sure why the react componet (and angular wrapper) is not being used, think for the annoations & templating use cases it did not work as we need custom body with help box but for the others not sure.

@torkelo the new component accepts `message`/`footer` props that can be a React elements so we can achieve than

Comment (dprokop) [reply public/sass/components/_empty_list_cta.scss]: @torkelo created followup issue as this is not in the scope of this PR #16248. Also, brought the sass file back untill #16248 is done.

Comment (dprokop) [reply public/sass/components/_empty_list_cta.scss]: @Ijin08 could you please re-check this?

**Review by Ijin08 — APPROVED** [review]
The empty list CTA seems to look fine now.

**Review by torkelo — APPROVED** [review]

**Pull request merged**